RandomExchangeOfSubstings/EntryPoint.cs

```
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace RandomExchangeOfSubstings
+{
+    /// <summary>
+    /// Used only as an entry point to the program
+    /// </summary>
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                char[] firstLine = Console.ReadLine().ToCharArray();
+                char[] secondLine = Console.ReadLine().ToCharArray();
+
+                SubstringManipulator manipulator = new SubstringManipulator();
+
+                manipulator.ExchangeRandomSubstrings(ref firstLine, ref secondLine);
+
+                Console.WriteLine(firstLine);
+                Console.WriteLine(secondLine);
+            }
```

Empty catch, what for? you have at least log error